src/assets/illustrations/fallback/fallback_code.svg

```diff
@@ -0,0 +1,76 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="800" height="450" viewBox="0 0 800 450">
```

Comment:
Is the `xmlns:xlink` attribute necessary now that masks are removed? (Feedback applies to all images)